src/dotnet/commands/dotnet-sln/add/Program.cs

```diff
@@ -32,7 +34,32 @@ internal class AddProjectToSolutionCommand : CommandBase
             _appliedCommand = appliedCommand;
 
             _fileOrDirectory = fileOrDirectory;
-            _inRoot = appliedCommand.ValueOrDefault<bool>(InRootOption);
+
+            var inRoot = appliedCommand.ValueOrDefault<bool>(InRootOption);
+            var relativeRoot = _appliedCommand.ValueOrDefault<string>(SolutionFolderOption);
```

Comment:
I think we should error (i.e. throw a `GracefulException`) if both options are specified since they are mutually exclusive.